options.html

```diff
@@ -1,162 +1,128 @@
-            <option value="yes">yes</option>
-        </select>
-    <p>
-
-    <hr>
-
-    <p>
-        The <b>Show Grammar and Usage Notes</b> setting allows you to turn on grammar
-        and usage notes.
-    </p>
-    <p>
-        <label for="grammar">Show grammar and usage notes:</label>
-        <select id="grammar" name="grammar">
-            <option value="yes">yes</option>
-            <option value="no">no</option>
-        </select>
-    </p>
-
-    <hr>
-
-    <p>
-        The <b>Save to Word List</b> setting allows you to specify what
-        to save to the internal word list.
-    </p>
-    <p>
-        <label for="saveToWordList">Save to word list:</label>
-        <select id="saveToWordList" name="saveToWordList">
-            <option value="allEntries">all entries</option>
-            <option value="firstEntryOnly">only the first entry</option>
-        </select>
-    </p>
-
-    <hr>
-
-    <p>
-        The <b>Pop-Up Color</b> setting allows you to change the background
-        color of the window showing the dictionary entries. (Changing this setting requires reloading the page
-        with the pop-up to take effect.)
-    </p>
-    <p>
-        <label for="popupcolor">Pop-up color:</label>
-        <select id="popupcolor" name="popupcolor">
-            <option value="yellow">Yellow</option>
-            <option value="blue">Blue</option>
-            <option value="lightblue">Light Blue</option>
-            <option value="black">Black</option>
-        </select>
-    </p>
-
-    <hr>
-
-    <p>
-        The <b>Skritter Domain</b> setting allows you to change the URL that is
-        used for adding words to your Skritter queue (Skritter login required).
-        By default, <code>skritter.com</code> is being used, but for some users
-        <code>skritter.cn</code> might work better.
-    </p>
-    <p>
-        <label for="skritterTLD">Skritter domain:</label>
-        <select id="skritterTLD" name="skritterTLD">
-            <option value="com">skritter.com</option>
-            <option value="cn">skritter.cn</option>
-        </select>
-    </p>
-
-    <hr>
-
-    <span class="important">Don't forget to save your changes by clicking on the botton at the top of this page.</span>
-</form>
-
+    <div class="container">
+        <h1 class="my-3">Options for the Zhongwen Browser Extension</h1>
+
+        <h2 class="mb-3">Pop-up Options</h2>
+
+        <div class="form-group mb-4">
+            <label for="popupcolor">
+                Background color of the pop-up window.
+                <small class="form-text text-muted">Changing this setting requires reloading the page with the pop-up to take effect.</small>
+            </label>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="popupColorYellow" name="popupColor" class="custom-control-input">
+                <label class="custom-control-label" for="popupColorYellow">Yellow pop-up background.</label>
```

Comment:
Maybe it's better not to add a period at the end of those options that do not form a grammatically complete sentence.